test cases/unit/123 pkgsubproj/meson.build

```diff
@@ -0,0 +1,3 @@
+project('pkg_opt_test', 'c')
```

Comment:
We can omit the language here and avoid performing compiler initialization for this unittest.

Reply:
(and in the subproject)